src/partial-templates/partial-templates.controller.ts

```diff
@@ -1,20 +1,30 @@
-import { Controller, Get, Post, Body, Patch, Param, Delete, ParseIntPipe, UseGuards } from '@nestjs/common';
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+  ParseIntPipe,
+  UseGuards,
+} from '@nestjs/common';
 import { PartialTemplatesService } from './partial-templates.service';
 import { CreatePartialTemplateDto } from 'src/models/partialTemplate/create-partial-template.dto';
 import { UpdatePartialTemplateDto } from 'src/models/partialTemplate/update-partial-template.dto';
 import { JwtAuthGuard } from 'src/auth/strategies/guards/jwt-auth.guard';
 
 @Controller('partial-templates')
 export class PartialTemplatesController {
+  constructor(
+    private readonly partialTemplatesService: PartialTemplatesService,
+  ) {}
 
-  constructor(private readonly partialTemplatesService: PartialTemplatesService) {}
-
-  
   /**
-  * Creates a new partialTemplate
-  * @param {CreatePartialTemplateDto} createPartialTemplateDto The partialTemplate data transfer object
-  * @returns - The created partialTemplate
-  */
+   * Creates a new partialTemplate
+   * @param {CreatePartialTemplateDto} createPartialTemplateDto The partialTemplate data transfer object
+   * @returns - The created partialTemplate
+   */
   @Post()
   create(@Body() createPartialTemplateDto: CreatePartialTemplateDto) {
```

Comment:
### I consider it necessary to update this type of methods to:
```ts
@Post()
 create(@Body() createAreaDto: CreateAreaDto) {
 return this.areasService.create(createAreaDto);
 }
```
### For reasons of
- This approach is clear and simple. The DTO is taken directly and passed to the service.
- It is easy to maintain because any change to the DTO only requires a change in one place.
- If the DTO is updated, this method automatically reflects those changes without the need for additional modifications.